EmulatorPkg/Win/Host/WinHost.c

```diff
@@ -1131,15 +1131,9 @@ PeCoffLoaderRelocateImageExtraAction (
 
     if ((Library != NULL) && (DllEntryPoint != NULL)) {
       Status = AddModHandle (ImageContext, Library);
-      if (Status == EFI_ALREADY_STARTED) {
+      if ((Status == EFI_SUCCESS) || (Status == EFI_ALREADY_STARTED)) {
         //
-        // If the DLL has already been loaded before, then this instance of the DLL can not be debugged.
-        //
-        ImageContext->PdbPointer = NULL;
-        SecPrint ("WARNING: DLL already loaded.  No source level debug %S.\n\r", DllFileName);
-      } else {
-        //
-        // This DLL is not already loaded, so source level debugging is supported.
+        // This DLL is either not loaded or already started, so source level debugging is supported.

```

Comment:
Old logic was there for a reason.
I am not sure if it's safe to just use the DllEntryPoint for a loaded module.

Reply:
i checked the history of this code, and it introduced in the below check-in 
https://github.com/tianocore/edk2/commit/10ccc27c9548ab2550689e68f6f45dc48120bc79

Based on the description 
when the Driver is unloaded then it should remove the handle.

that we aren't changing from this PR.

may be i might i missed the complete picture of the PR where it introduced, can you check that PR once please